src/Classes/ServiceAPI/MyRadio_BannerCampaign.php

```diff
@@ -168,9 +168,9 @@ public static function create(
         $campaign = self::getInstance($result[0]);
 
         foreach ($timeslots as $timeslot) {
-            $campaign->addTimeslot($timeslot['day'], $timeslot['start_time'], $timeslot['end_time']);
+            $campaign->addTimeslot($timeslot['day'], $timeslot['start_time'], $timeslot['end_time'], 'order',$result[0]);
```

Comment:
is there an issue with '"order"' vs 'order' here? (i'd prefer to keep the code the same and change the column name instead)